README.rst

```diff
@@ -1,4 +1,4 @@
-Fedora MirrorManager
+Fedoras MirrorManager
```

Comment:
Not sure if this is correct... Shouldn't this be `Fedora's MirrorManager`?

Reply:
my apoligies, i did this late last night to try to see if the rawhide tests in CI were failing on a mostly clean run. Forgot to mark it as WIP.